### DISCUSSION TITLE
Error deleting file

### DISCUSSION THREAD

**Discussion by EpicJosch** [nextcloud/all-in-one#2691] [Questions] [answered]
### Steps to reproduce
1. Create folder
2. delete folder
3. error message appears

### Expected behavior 
The folder should get deleted

### Actual behavior 
The folder is not deleted

### Host OS 
LINUX (Ubuntu)

#### Nextcloud AIO version 
26.0.2

#### Current channel 
latest

#### Other valuable info 

![grafik](https://github.com/nextcloud/all-in-one/assets/50040609/a64c72a0-10c8-4fc9-a653-177033b55409)

**Comment by EpicJosch** [comment]
The file in the screenshot is a test file. (German Nextcloud System as you can see)

**Comment by szaimen** [comment, marked as answer]
Hi, do you have the music app installed? Can yiu check if disabling it makes things work?

**Reply by EpicJosch** [reply]
Yes works now.. thanks.. any idea whats the problem here?

**Reply by szaimen** [reply]
https://github.com/owncloud/music/issues/1068
